setup.py

```diff
@@ -66,5 +66,4 @@
 from setuptools import setup  # noqa
 from extension_helpers import get_extensions  # noqa
 
-setup(use_scm_version={'write_to': os.path.join('astropy', '_version.py')},
-      ext_modules=get_extensions())
+setup(use_scm_version=True, ext_modules=get_extensions())
```

Comment:
Kinda a shame you still need `use_scm_version=True` at all really.

Reply:
I just ran a local `tox -e test` without it. Appears to work!

Reply:
And appears to work here.